src/test/java/uk/gov/hmcts/cmc/claimstore/services/ClaimantCCJRequestServiceTest.java

```diff
@@ -0,0 +1,125 @@
+                    .builder().build())
+                .build();
+            pdf = new PDF(
+                buildClaimIssueReceiptFileBaseName(claim.getReferenceNumber()),
+                PDF_CONTENT,
+                CLAIM_ISSUE_RECEIPT
+            );
+            when(ccjByAdmissionOrDeterminationPdfService.createPdf(claim)).thenReturn(pdf);
+            when(documentService.uploadToDocumentManagement(any(PDF.class),
+                anyString(), any(Claim.class))).thenReturn(claim);
+            event = new CountyCourtJudgmentEvent(claim, "authorisation");
+            service.getAndSaveDocumentToCcd(event);
+            verify(ccjByAdmissionOrDeterminationPdfService)
+                .createPdf(claim);
+            verify(documentService)
+                .uploadToDocumentManagement(pdf, USER.getAuthorisation(), claim);
+        }
+
+        @Test
+        void shouldSaveClaimDocumentToCcdIfByDetermination() {
+            CountyCourtJudgment ccj = CountyCourtJudgment.builder()
+                .ccjType(CountyCourtJudgmentType.DETERMINATION).build();
+            claim = SampleClaim.builder()
+                .withCountyCourtJudgment(ccj)
+                .withClaimantResponse(SampleClaimantResponse.ClaimantResponseAcceptation
+                    .builder().build())
+                .build();
+            pdf = new PDF(
+                buildClaimIssueReceiptFileBaseName(claim.getReferenceNumber()),
+                PDF_CONTENT,
+                CLAIM_ISSUE_RECEIPT
+            );
+            when(ccjByAdmissionOrDeterminationPdfService.createPdf(claim)).thenReturn(pdf);
+            when(documentService.uploadToDocumentManagement(any(PDF.class),
+                anyString(), any(Claim.class))).thenReturn(claim);
+            event = new CountyCourtJudgmentEvent(claim, "authorisation");
+            service.getAndSaveDocumentToCcd(event);
+            verify(ccjByAdmissionOrDeterminationPdfService)
+                .createPdf(claim);
+            verify(documentService)
+                .uploadToDocumentManagement(pdf, USER.getAuthorisation(), claim);
+        }
+
+        @Test
+        void shouldNotSaveClaimDocumentToCcdIfByDefault() {
+            CountyCourtJudgment ccj = CountyCourtJudgment.builder()
+                .ccjType(CountyCourtJudgmentType.DEFAULT).build();
+            claim = SampleClaim.builder()
+                .withCountyCourtJudgment(ccj)
+                .withClaimantResponse(SampleClaimantResponse.ClaimantResponseAcceptation
+                    .builder().build())
+                .build();
+            event = new CountyCourtJudgmentEvent(claim, "authorisation");
+            service.getAndSaveDocumentToCcd(event);
+            verify(ccjByAdmissionOrDeterminationPdfService, never())
+                .createPdf(claim);
+            verify(documentService, never())
+                .uploadToDocumentManagement(pdf, USER.getAuthorisation(), claim);
```

Comment:
Normally, verifications should be as specific as possible... except for `never()` where they should be as general as possible. Recommend using the `any(clazz)` methods